hamnosys_dicts.py

```diff
@@ -0,0 +1,238 @@
+DescriptionDict = {
+    "1": "name",
+    "2": "start",
+    "3": "end",
+    "4": "Symmetry operator",
+    "5": "Dom hand - Handshape - base form",
+    "6": "Dom hand - Handhape - Thumb position",
+    "7": "Dom hand - Handshape - Bending",
+    "8": "Dom hand - Handposition - Extended finger direction",
+    "9": "Dom hand - Handposition - Palm orientation",
+    "10": "Dom hand - Handlocation - Frontal plane l/r",
+    "11": "Dom hand - Handlocation - Frontal plane t/b",
+    "12": "Dom hand - Handlocation - Distance from the body",
+}
+
+HandlocationFronalPlaneRL = {
+    "0": "",  # left to the left
+    "1": "",  # left
+    # "2" : "", #center
+    "3": "",  # right
+    "4": "",  # right to the right
+}
+
+HandLocationFronalPlaneTB = {
+    #   "0" : None,
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+    "6": "",
+    "7": "",
+    "8": "",
+    "9": "",
+    "10": "",
+    "11": "",
+    "12": "",
+    "13": "",
+    "14": "",
+    "15": "",
+    "16": "",
+    "17": "",
+    "18": "",
+    "19": "",
+    "20": "",
+    "21": "",
+    "22": "",
+    "23": "",
+    "24": "",
+    "25": "",
+    "26": "",
+    "27": "",
+    "28": "",
+    "29": "",
+    "30": "",
+    "31": "",
+    "32": "",
+    "33": "",
+    "34": "",
+    "35": "",
+    "36": "",
+}
+
+HandshapeBaseformsDict = {
+    "0": "",
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+    "6": "",
+    "7": "",
+    "8": "",
+    "9": "",
+    "10": "",
+    "11": "",
+}
+
+HandshapeBendingDict = {
+    #   "0" : None,
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+}
+
+HandpositionFingerDirectionDict = {
+    "0": "",
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+    "6": "",
+    "7": "",
+    "8": "",
+    "9": "",
+    "10": "",
+    "11": "",
+    "12": "",
+    "13": "",
+    "14": "",
+    "15": "",
+    "16": "",
+    "17": "",
+}
+
+HandpositionPalmOrientationDict = {
+    "0": "",
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+    "6": "",
+    "7": "",
+}
+
+HandshapeThumbPositionDict = {
+    #   "0" : None
+    "1": "",
+    "2": "",
+    "3": "",
+}
+
+HandLocationDistanceDict = {
+    "0": "",
+    "1": "",
+    "2": "",
+    #   "3" : None
```

Comment:
and here

Reply:
maybe miss of the feature should always be at the same position - 0?

Reply:
Agree